examples/compare_runs.py

```diff
@@ -0,0 +1,347 @@
+between two `examples/` folders.
+
+However, this script can also be used to compare two run folders directly.
+
+For more information read 'docs/integration_tests.md'.
+
+## USAGE
+
+To obtain help:
+
+`python compare_runs.py -h`
+
+To compare two `examples/` folders:
+
+`python compare_runs.py -d <PATH-TO-TESTS> -r <PATH-TO-REFERENCE>`
+
+For example, if you are inside the development repository's examples folder,
+you need to give `-d` the current path:
+
+`python compare_runs.py -d . -r /home/user/haddock3main/examples`
+
+To compare two specific run folders:
+
+`python compare_runs.py -r1 <PATH-TO-RUN1> -r2 <PATH-TO-RUN2>`
+
+For example:
+
+`python compare_runs.py -r1 scoring/run1-test -r2 scoring/run2-test`
+
+Paramters `-d` and `-r` and mutually exclusive from `-r1` and `r2`.
+"""
+import argparse
+import csv
+import os
+import sys
+from functools import partial
+from math import isclose
+from pathlib import Path
+
+
+try:
+    from haddock.gear.config_reader import read_config
+except Exception:
+    print(  # noqa: T001
+        "Haddock3 could not be imported. "
+        "Please activate the haddock3 python environment.",
+        file=sys.stderr,
+        )
+    sys.exit(1)
+
+
+printf = partial(print, flush=True)  # noqa: T002
+
+
+examples = (
+    ("docking-antibody-antigen"    , "docking-antibody-antigen-ranairCDR-test.cfg"),  # noqa: E203, E501
+    ("docking-antibody-antigen"    , "docking-antibody-antigen-ranairCDR-clt-test.cfg"),  # noqa: E203, E501
+    ("docking-antibody-antigen"    , "docking-antibody-antigen-CDR-accessible-test.cfg"),  # noqa: E203, E501
+    ("docking-antibody-antigen"    , "docking-antibody-antigen-CDR-accessible-clt-test.cfg"),  # noqa: E203, E501
+    ("docking-protein-DNA"         , "docking-protein-DNA-test.cfg"),  # noqa: E203, E501
+    ("docking-protein-DNA"         , "docking-protein-DNA-mdref-test.cfg"),  # noqa: E203, E501
+    ("docking-protein-homotrimer"  , "docking-protein-homotrimer-test.cfg"),  # noqa: E203, E501
+    ("docking-protein-ligand-shape", "docking-protein-ligand-shape-test.cfg"),  # noqa: E203, E501
+    ("docking-protein-ligand"      , "docking-protein-ligand-test.cfg"),  # noqa: E203, E501
+    ("docking-protein-peptide"     , "docking-protein-peptide-test.cfg"),  # noqa: E203, E501
+    ("docking-protein-peptide"     , "docking-protein-peptide-mdref-test.cfg"),  # noqa: E203, E501
+    ("docking-protein-protein"     , "docking-protein-protein-test.cfg"),  # noqa: E203, E501
+    ("docking-protein-protein"     , "docking-protein-protein-cltsel-test.cfg"),  # noqa: E203, E501
+    ("docking-protein-protein"     , "docking-protein-protein-mdref-test.cfg"),  # noqa: E203, E501
+    ("refine-complex"              , "refine-complex-test.cfg"),  # noqa: E203, E501
+    ("scoring"                     , "emscoring-test.cfg"),  # noqa: E203, E501
+    ("scoring"                     , "mdscoring-test.cfg"),  # noqa: E203, E501
+    )
```

Comment:
I am a bit skeptical about hard-coding all these examples here, as we have them hard-coded in another script (`run_tests.py`). Wouldn't be better to create a list and import it in both cases? so that when we add a an example we only need to modify the file with the list

Reply:
Yes, you are right. These were hard coded at first when we were prototyping them. But now it makes little sense. I will arrange that in another pull request.